app/scripts/HiGlassComponent.js

```diff
@@ -3074,7 +3074,8 @@ class HiGlassComponent extends React.Component {
     if (
       track.type === 'viewport-projection-center' ||
       track.type === 'viewport-projection-horizontal' ||
-      track.type === 'viewport-projection-vertical'
+      track.type === 'viewport-projection-vertical' ||
+      (track.options && track.options.addCallbacks)
```

Comment:
This might be a problem because a track's options are set by the user and if they remove the `addCallbacks` section then it'll stop working. You could maybe use the track's datatype setting (see `tracks-info.js`)?

Reply:
Thank you for this suggestion! I've changed to use `config.projection` instead, which is not visible to users.